pvlib/pvsystem.py

```diff
@@ -46,16 +49,6 @@
 }
 
 
-TEMP_MODEL_PARAMS = {
```

Comment:
this change should also be documented in the API section. I'd say no need to deprecate its presence here, though.